pkg/debug/checkups/host.go

```diff
@@ -38,12 +40,21 @@ func (hc *hostInfoCheckup) Run(ctx context.Context, extraFH io.Writer) error {
 	desktopProcesses := runner.InstanceDesktopProcessRecords()
 	hc.data["user_desktop_processes"] = desktopProcesses
 
+	uptimeRaw, err := host.Uptime()
+	if err != nil {
+		hc.data["uptime_friendly"] = fmt.Sprintf("ERROR: %s", err.Error())
+	} else {
+		hc.data["uptime_friendly"] = formatUptime(uptimeRaw)
+	}
+
+	hc.data["uptime"] = uptimeRaw
```

Comment:
Shouldn't this be inside that `err == nil`

Reply:
i assumed it would just be zero value and the friendly would show the error. I can leave unset in that case if you'd like

Reply:
I'm never really sure if the return is defined if there's an error. So my bias would be to drop it. (it should be zero, but...)